x-pack/plugins/security_solution/server/routes/index.ts

```diff
@@ -178,6 +179,7 @@ export const initRoutes = (
   assetCriticalityGetRoute(router, logger);
   assetCriticalityDeleteRoute(router, logger);
   assetCriticalityPrivilegesRoute(router, getStartServices, logger);
+  assetCriticalityCSVUploadRoute(router, logger, config, getStartServices);
```

Comment:
While it's possible to register individual routes here it's much better to incapsulate the details. I recommend to move all `assetCriticality...` route registrations into `registerAssetCriticalityRoutes()` function. This function file should have explicit ownership.